src/app/components/reports-page/reports-page.component.html

```diff
@@ -81,7 +81,7 @@
             </ul>
           </div>
 
-          <div *ngIf="selFormatFilter == 'Code' || selFormatFilter == 'All'"
+          <div *ngIf="selFormatFilter.includes('Code')||selFormatFilter.includes('All')"
```

Comment:
Seems odd to have a filter that could have both "Code" and "All", why not just remove "All" as an option? Same point as other similar comment.

Reply:
How is this resolved? Is the filter supposed to be able to have 'Code' and 'All' at the same time? I understand the reports page is structured a bit differently than the other pages, but could this be fixed? Or does it need to be left as is for some reason?

Reply:
For now it needs to remain as is.  The reports page has an entirely different implementation and filter methods on the backend from the Content Finder and Request Finder (why they didn't use the same filter methods across the board I have no idea.)  That being said, it is not feasible to gut all of that and change it within the next 24 hours while others are waiting on this pull to add their features.  I intend to create a new task for that feature and create a branch for it and will try and get it working tonight but others need this in now.